src/components/TraineeAttendance.tsx

```diff
@@ -1,15 +1,82 @@
 /* eslint-disable */
-import React from 'react';
+import React, { useState } from 'react';
 import { useTranslation } from 'react-i18next';
-import { Link } from 'react-router-dom';
+import { useQuery } from '@apollo/client';
 import Pagination from '../components/Pagination';
 import AttendanceData from '../dummyData/attendance.json';
 import useDocumentTitle from '../hook/useDocumentTitle';
-import Button from './Buttons';
+import { GET_WEEKLY_ATTENDANCE } from '../Mutations/Attendance';
 
-const TraineeAttendance = () => {
+interface TraineeAttendanceData {
+  getTraineeAttendanceByID: {
+    weekNumber: string;
```

Comment:
Wouldn't this be of type number logically?

Reply:
yeah, and I also found it weird. I also looked into it & found that whoever creates attendance, used a string to save week number.